interview-questions.md

```diff
@@ -8,62 +8,72 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is iteration?
 
-  Your answer:
+  Your answer: This word describes how many times something does Something a good example of this is how many times The wheels on a car rotate. 
+  Whenever I hear the word iteration I always think of being in the Army and how many push-ups we are about to do, so this word really does not have any pleasant memories for me.
```

Comment:
Yea, like how each set of push ups is a iterating over the number of push-ups you are doing in that set.